doc/inifile.rst

```diff
@@ -490,15 +490,16 @@ The following options are not recognized unless "nat" is set.
 
 * nat-ip (string: IP address)
 
-  ??
+  Target IP address for the tunnelling servers responses and requests.
+  When set to zero the server shall use the IP address in the IP package received.
```

Comment:
Proposal: Replace "package" with "packet".

```suggestion
  When set to zero the server shall use the IP address in the IP packet received.
```

Reply:
Fine for me. I was copying this from KNX specs.